content/applications/websites/elearning.rst

```diff
@@ -0,0 +1,258 @@
+- :guilabel:`Documentation`: the content is available any order. If you chose this option, you can
+  select the :guilabel:`Featured Content` on the course homepage.
+
+.. _elearning/karma:
+
+Karma tab
+~~~~~~~~~
+
+This tab is about gamification to make e-learning fun and interactive.
+
+In the :guilabel:`Rewards` section, set up the rewards you want to grant your students when they
+:guilabel:`Review` or :guilabel:`Finish` a course.
+
+In the :guilabel:`Access Rights` section, define the karma needed to :guilabel:`Add Review`,
+:guilabel:`Add Comment`, or :guilabel:`Vote` for the course.
+
+.. note::
+   From your course, use the smart button :guilabel:`Contact attendees` to reach people who are
+   already part of the course.
+
+.. _elearning/create-content:
+
+Content
+-------
+
+Manage your content by going to :menuselection:`eLearning --> Courses --> Contents`. Click
+:guilabel:`New` to create content. Add your :guilabel:`Content Title`, and if you want
+:ref:`Tags <elearning/tags>`, then fill in the related information among the different tabs.
+
+.. image:: elearning/elearning-content-tab.png
+   :align: center
+   :alt: Create your content.
+
+Document tab
+~~~~~~~~~~~~
+
+- :guilabel:`Course`: select the course your content belongs to;
+- :guilabel:`Content Type`: select your type of content;
+- :guilabel:`Responsible`: add a responsible person for your content;
+- :guilabel:`Duration`: indicate the duration required to complete the course;
+- :guilabel:`Allow Download`: allow users to download the content of the slide. This option is only
+  visible when the content is a document;
+- :guilabel:`Allow Preview`: the course is accessible by anyone, users don’t need to join the
+  channel to access the content of the course;
+- :guilabel:`# of Public Views`: it represents the views of the non-attending participants;
+- :guilabel:`# Total Views`: it represents the total of the public views and the attendees views.
+
+Description tab
+~~~~~~~~~~~~~~~
+
+You can add a description of your content. It appears front end in the :guilabel:`About` section of
+your course content.
+
+Additional resources tab
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`Add a line` to add links or resources that benefit your participants. They appear
+in the course content on your website.
+
+.. image:: elearning/additional-content.png
+  :align: center
+  :alt: Additional ressources
+
+Quiz tab
+~~~~~~~~
+
+From this tab you can create a quiz to assess your students at the end of the course.
+
+Fill in the :guilabel:`Points Rewards`. They are offered to the participant based on the number of
+trials to correctly answer the question, and keep them engaged. Then, create your
+:guilabel:`Questions` and :guilabel:`Answers` by clicking on :guilabel:`Add a line`. A new window
+pops up. Fill in the :guilabel:`Question Name` and add multiple answers by clicking on
+:guilabel:`Add a line`. Tick the box for the correct answer. You can also fill in the
```

Comment:
A new window pops up, and you can fill in the Question Name and add multiple answers by clicking on Add a line.